sdk/src/genesis_block.rs

```diff
@@ -33,6 +39,7 @@ pub struct GenesisBlock {
     pub rent_calculator: RentCalculator,
     pub inflation: Inflation,
     pub epoch_schedule: EpochSchedule,
+    pub operating_mode: OperatingMode,
```

Comment:
should we expose this to developers (like make it a sysvar)?

Reply:
Can you think of a use case for why that'd be helpful?  I don't see a strong need right now

Reply:
only https://github.com/solana-labs/solana/issues/6193

Reply:
no use cases, just a general issue of programs not being able to see network config

Reply:
🤷‍♂ This is "meta config".  A program that wants to do something based on OperatingMode is probably completely broken